docs/sphinx/user/inputs.rst

```diff
@@ -1166,3 +1168,26 @@ Example::
 
 The first line of the file contains the total number of probes for this set.
 This is followed by the coordinates (three real numbers), one line per probe.
+
+Total Kinetic Energy 
+`````````````````````
+
+The ``KineticEnergy`` sampling integrates the kinetic energy over the domain 
+and outputs it to a text file identified using the label and output in 
+the post_processing directory.
+Example::
+
+  sampling.ke1.type              = KineticEnergy
+  sampling.ke2.output_frequency  = 2
+
+Total Enstrophy
+`````````````````
+
+The ``Enstrophy`` sampling integrates the enstrophy over the domain and
+outputs it to a text file identified using the label and output in 
+the post_processing directory.
+
+Example::
+
+  sampling.enst1.type              = Enstrophy
```

Comment:
Why is there a sampling prefix here? Should this be the keyword provided in postprocessing section?

Reply:
oh good catch, I'll fix it